tests/test_admin.py

```diff
@@ -616,7 +671,7 @@ def test_menuitem_move_node_smoke_test(self):
 
     @patch("django.contrib.messages.error")
     def test_menuitem_move_node_cant_move_outside_of_root(self, mocked_messages):
```

Comment:
"""A node can't be moved outside of a root node because that would pull it out of a tree altogether
"""